buildscripts/make_dependencies.sh

```diff
@@ -55,6 +55,8 @@ else
       GCC_ARCH=s390x-linux-gnu
     elif [[ "$ARCH" == loongarch_64 ]]; then
       GCC_ARCH=loongarch64-unknown-linux-gnu
+    elif [[ "$ARCH" == riscv* ]]; then
+      GCC_ARCH=riscv64-unknown-linux-gnu
```

Comment:
This should be `riscv64-linux-gnu`  for the multiarch container. Running a Ubuntu 18.04 container on my machine shows that riscv64-linux-gnu-gcc exists. I think that will fix the CI failure.